resources/MSBuildImports/15.0/SolutionFile/ImportAfter/Microsoft.NuGet.ImportAfter.targets

```diff
@@ -0,0 +1,18 @@
+<!--
```

Comment:
This is similar to the project level ImportAfter target already in the CLI. Under the SolutionFile folder it is applied to the metaproj generated from a sln file. 

https://github.com/Microsoft/msbuild/issues/920#issuecomment-250495107